UI/profile-card.tsx

```diff
@@ -0,0 +1,55 @@
+/** @jsx h */
+import { Component } from "https://esm.sh/preact@10.17.1";
+import { h } from "https://esm.sh/preact@10.17.1";
+import { tw } from "https://esm.sh/twind@0.16.16";
+import { emitFunc } from "../event-bus.ts";
+import { ProfileData } from "../features/profile.ts";
+import { PublicKey } from "../lib/nostr-ts/key.ts";
+import { Avatar } from "./components/avatar.tsx";
+import { DividerClass } from "./components/tw.ts";
+import { ViewUserDetail } from "./message-panel.tsx";
+import { HoverButtonBackgroudColor, PrimaryTextColor } from "./style/colors.ts";
+
+type Props = {
+    profileData?: ProfileData;
+    publicKey: PublicKey;
+    emit: emitFunc<ViewUserDetail>;
+};
+
+export class ProfileCard extends Component<Props, {}> {
```

Comment:
We should not use class component if this component has no inernal mutable states.